### PR TITLE
Hotfix/fesom cmake specific

### PR DESCRIPTION
**Pull request by mandresm**
Same as #206 with some extra modifications for AWICMMOD.

### CODE REVIEW

**Comment by mandresm** [comment]
@nwieters , could you please check if you are happy with the modifications I made on the coupling files for `AWICMMOD`?

File: configs/couplings/fesom-2.0-esm_interface-oasis+echam-6.3.04p1-esm_interface-oasis/fesom-2.0-esm_interface-oasis+echam-6.3.04p1-esm_interface-oasis.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+components:
+- oasis3mct-2.8
+- esm_interface-generic
+- echam-6.3.04p1-esm_interface
+- fesom-2.0-esm_interface
+coupling_changes:
+- sed -i '/set(FESOM_COUPLED/s/OFF/ON/g' fesom-1.4/CMakeLists.txt
```
Comment (nwieters) [inline configs/couplings/fesom-2.0-esm_interface-oasis+echam-6.3.04p1-esm_interface-oasis/fesom-2.0-esm_interface-oasis+echam-6.3.04p1-esm_interface-oasis.yaml]: `- sed -i '/set(FESOM_COUPLED/s/OFF/ON/g' fesom-2.0/CMakeLists.txt`

File: configs/couplings/fesom-2.0-esm_interface-yac+echam-6.3.04p1-esm_interface-yac/fesom-2.0-esm_interface-yac+echam-6.3.04p1-esm_interface-yac.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+components:
+- oasis3mct-2.8
+- yac-1.5.4
+- esm_interface-generic
+- echam-6.3.04p1-esm_interface
+- fesom-2.0-esm_interface
+coupling_changes:
+- sed -i '/set(FESOM_COUPLED/s/OFF/ON/g' fesom-1.4/CMakeLists.txt
```
Comment (nwieters) [inline configs/couplings/fesom-2.0-esm_interface-yac+echam-6.3.04p1-esm_interface-yac/fesom-2.0-esm_interface-yac+echam-6.3.04p1-esm_interface-yac.yaml]: `- sed -i '/set(FESOM_COUPLED/s/OFF/ON/g' fesom-2.0/CMakeLists.txt`

**Review by nwieters — CHANGES_REQUESTED** [review]
Hi, 
I corrected two `sed` commands that used `fesom-1.4` instead of `fesom-2.0 ` folder.

File: configs/couplings/fesom-2.0-esm_interface-yac+echam-6.3.04p1-esm_interface-yac/fesom-2.0-esm_interface-yac+echam-6.3.04p1-esm_interface-yac.yaml (hunk 2 above)
Comment (mandresm) [reply configs/couplings/fesom-2.0-esm_interface-yac+echam-6.3.04p1-esm_interface-yac/fesom-2.0-esm_interface-yac+echam-6.3.04p1-esm_interface-yac.yaml]: Solved

File: configs/couplings/fesom-2.0-esm_interface-oasis+echam-6.3.04p1-esm_interface-oasis/fesom-2.0-esm_interface-oasis+echam-6.3.04p1-esm_interface-oasis.yaml (hunk 1 above)
Comment (mandresm) [reply configs/couplings/fesom-2.0-esm_interface-oasis+echam-6.3.04p1-esm_interface-oasis/fesom-2.0-esm_interface-oasis+echam-6.3.04p1-esm_interface-oasis.yaml]: Solved

**Review by nwieters — APPROVED** [review]
Thank you

**Pull request merged**